utils/build.ps1

```diff
@@ -3446,6 +3447,21 @@ function Build-Subprocess([Hashtable] $Platform) {
     }
 }
 
+function Build-ToolsProtocols([Hashtable] $Platform) {
+  $SDKROOT = Get-SwiftSDK -OS $Platform.OS -Identifier $Platform.DefaultSDK
```

Comment:
Why not inline the single use variable?

Reply:
no particular reason - I can update this once I get results from the latest round of CI builds